### PR TITLE
Manifest v2

### PR DESCRIPTION
**Pull request by gabriel-okteto**

### CODE REVIEW

File: okteto.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: java-maven-getting-started
```
Comment (pchico83) [inline okteto.yml]: remove `name`

File: okteto.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+build:
+  java-maven-getting-started:
+    context: .
+    dockerfile: Dockerfile
```
Comment (pchico83) [inline okteto.yml]: remove `dockerfile`

File: okteto.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: java-maven-getting-started
+build:
+  java-maven-getting-started:
+    context: .
```
Comment (pchico83) [inline okteto.yml:4]: add `image: okteto.dev/java-maven-hello-world:1.0.0`

File: okteto.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/maven:3
+    command: java -jar *.jar
```
Comment (pchico83) [inline okteto.yml]: `command: mvn spring-boot:run`

File: okteto.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    workdir: /app
```
Comment (pchico83) [inline okteto.yml]: remove `workdir`

File: okteto.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    sync:
+      - .:/app
```
Comment (pchico83) [inline okteto.yml]: `.:/usr/src/app`

File: okteto.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    forward:
+      - 5005:5005
+      - 8080:8080
```
Comment (pchico83) [inline okteto.yml]: remove `8080`

**Review by pchico83 — COMMENTED** [review]
@gabriel-okteto you are missing the changes to `k8s.yml`

File: okteto.yml (hunk 4 above)
Comment (gabriel-okteto) [reply okteto.yml]: ? in the node sample we default into back and not into npm run start, 
how is the java sample different?

Comment (gabriel-okteto) [reply okteto.yml]: ** bash

Comment (pchico83) [reply okteto.yml]: `mvn spring-boot:run` takes care of installing depedencies, but `npm run start` doesn't. you also need a different command for debugging, which makes a `bash` terminal more convenient for the nodejs sample

File: okteto.yml (hunk 6 above)
Comment (gabriel-okteto) [reply okteto.yml]: ? should I also update the Dockerfile then, 
the Dockerfile is staging the code to WORKDIR /app

Comment (pchico83) [reply okteto.yml]: no, we are using `okteto/maven:3`, which is not built by that Dockerfile

**Review by pchico83 — APPROVED** [review]

File: okteto.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+build:
+  hello-world:
+    image: okteto.dev/java-maven-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    command: mvn spring-boot:run
```
Comment (pchico83) [inline okteto.yml]: checking https://www.okteto.com/docs/samples/java/#maven-1, we are missing:
```
image: okteto/maven:3
command: bash
volumes:
  - /root/.m2
```

File: okteto.yml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+build:
+  hello-world:
+    image: okteto/maven:3
```
Comment (pchico83) [inline okteto.yml]: same as the gradle sample

**Review by pchico83 — APPROVED** [review]

**Pull request merged**